apps/mull-api/src/database/seeders/database.seeder.ts

```diff
@@ -0,0 +1,29 @@
+import { Connection } from 'typeorm';
+import { Factory, Seeder } from 'typeorm-seeding';
+import { Event } from '../../app/entities';
+
+/**
+ * Simple algorithm that will add the next 3 users as a friend
```

Comment:
No need for "Simple algorithm" just state what it does descriptively. I'm not sure what `add the next 3 users as a friend` means either

Reply:
Something like "For each user, add the next 3 users with greater id's as friends"